frontend/cypress/e2e/createANewProject.cy.ts

```diff
@@ -39,4 +39,15 @@ describe('Create a new project', () => {
     cy.contains('Crews Setting').should('exist')
     cy.contains('Real Done').should('exist')
   })
+
+  it('Should import a project manually', () => {
```

Comment:
Should create another test case named "importaProject.cy.ts"

Reply:
yes!